packages/x-tree-view/package.json

```diff
@@ -40,14 +40,13 @@
   },
   "dependencies": {
     "@babel/runtime": "catalog:",
+    "@base-ui-components/utils": "0.1.0",
```

Comment:
For now it's only used for a few utils
I'll do a follow up to migrate the other utils.
And in the future we will use the store from there too, once it's updated.